config/initializers/default_form_builder.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.action_view.default_form_builder = GOVUKDesignSystemFormBuilder::FormBuilder
```

Comment:
This is moving here so that it is loaded for view specs which don't load controllers. I think that allows us to better unit test our templates.